File: Patterns/js/pattern11.js
/*
n = 4

*
**
***
****

*/

function pattern(n) {
  let ans = "";

  let row = 1;

  while (row <= n) {
    let col = 1;

    while (col <= row) {
      ans += "*";
      col++;
    }

    ans += "\n";
    row++;
  }

  console.log(ans);
}

pattern(4);
pattern(10);

File: Patterns/go/pattern11.go
package main

import "fmt"

/*
n = 4

*
* *
* * *
* * * *

*/

func main() {
	var n int
	fmt.Scan(&n)

	row := 1
	for row <= n {
		col := 1

		for col <= row {
			fmt.Print("*")
			col++
		}
		
		fmt.Print("\n")
		row++
	}
}